cea/optimization/master/cost_model.py

```diff
@@ -125,8 +125,9 @@ def addCosts(indCombi, buildList, locator, dicoSupply, QUncoveredDesign, QUncove
         # Furnace
         if dicoSupply.Furnace_on == 1:
             P_design = dicoSupply.Furnace_Q_max
-            
-            fNameSlavePP = dicoSupply.configKey + "PPActivationPattern.csv"
+
+            fNameSlavePP = locator.optimization_slave_PPActivationPattern(dicoSupply.configKey)
```

Comment:
this is probably a bug. did you test? should be `locator.get_optimization_slave_PPActivationPattern(discoSupply.configkey)`...